addons-l10n/en/nfe-checker.json

```diff
@@ -0,0 +1,10 @@
+{
+  "nfe-checker/unreviewed": "Unreviewed",
+  "nfe-checker/title-unreviewed": "The Scratch Team has not looked at this project yet.",
+  "nfe-checker/fe": "Unrestricted",
+  "nfe-checker/title-fe": "The Scratch Team has reviewed this project and deemed it safe for everyone.",
+  "nfe-checker/nfe": "Restricted",
+  "nfe-checker/title-nfe": "This project reviewed by a Scratch Team member and while considered safe for Scratch, won't be available by searching for it and can't appear on the search bar. This rarely happens and it happens with reason. Do not create forum topics or otherwise try to get the Scratch Team to reconsider. If you want to ask why this decision happened, use the Contact Us link in the footer.",
+  "nfe-checker/unavailable": "Unavailable",
+  "nfe-checker/title-unavailable": "This project is not part of a remix tree (it has no remix and is not a remix)."
```

Comment:
It doesn't really make sense to label the text element "Unavailable" out of context (_the project is unavailable? Huh?_)

There's gotta be a better way to make it obvious that the status is unavailable...

Reply:
There's no longer an Unavailable option, so this is resolved